### ISSUE TITLE
Running vpu with xml configuration breaks for phpunit versions newer than 4.1.6

### ISSUE THREAD

**Issue by localgod** [VisualPHPUnit/VisualPHPUnit#131]

**Comment by Yanncha** [comment]
Hello !

While running with xml config, I get "Error Parsing Response From Server", and "Warning: Cannot modify header information - headers already sent by..."

I guess it is the same bug ? If needed I can send complete error message...
(with PHPUnit 4.6.9 installed with VPU's composer.json)

Best regards.

**Comment by localgod** [comment]
Could you please try this branch: https://github.com/VisualPHPUnit/VisualPHPUnit/tree/%23130
Let me know if that changes anything.

**Comment by Yanncha** [comment]
Ok, I can check.
However, I'm not very familiar with git/github. Can you help me on how to check out a given branch ?

Thanks.

**Comment by burzum** [comment]
I'm getting the same problem using my XML file. Running the master branch of VPU:

```
Warning (2): Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\VisualPHPUnit\vendor\phpunit\phpunit\src\Util\Printer.php:172) [C:\xampp\htdocs\VisualPHPUnit\nx\core\Response.php, line 164]
Code Context

header - [internal], line ??
nx\core\Response::render() - C:\xampp\htdocs\VisualPHPUnit\nx\core\Response.php, line 164
nx\core\Dispatcher::handle() - C:\xampp\htdocs\VisualPHPUnit\nx\core\Dispatcher.php, line 70
[main] - C:\xampp\htdocs\VisualPHPUnit\app\public\index.php, line 18
```

**Comment by kenjis** [comment]
I've got the same problem, and #143 works fine.

**Comment by localgod** [comment]
Fix with adfbe8c
